extensions/logging-json/runtime/src/main/java/io/quarkus/logging/json/runtime/AdditionalFieldConfig.java

```diff
@@ -16,7 +16,7 @@ public class AdditionalFieldConfig {
 
     /**
      * Additional field type specification.
-     * Supported types: string, int, long
+     * Supported types: `string`, `int`, and `long`.
```

Comment:
This makes no sense in the Javadoc. Either do this:

```suggestion
     * Supported types: {@code string}, {@code int}, and {@code long}.
```

or don't do anything at all.

Reply:
Was just trying to fix that issue of missing `.` and, additionaly, that oxford comma.
![image](https://github.com/quarkusio/quarkus/assets/48474054/170805cb-e4bd-4c0b-ab39-723b4f774b76)
The readability is not good atm.
Will apply your patch.